news/_posts/releases/2020-11-18-prestashop-1-7-7-0-available.md

```diff
@@ -0,0 +1,92 @@
+- [An error is displayed when trying to remove a voucher with free gift](https://github.com/PrestaShop/PrestaShop/issues/21810)
+- [The voucher line is not removed when the Order's total < Minimum amount for cart rule](https://github.com/PrestaShop/PrestaShop/issues/21812)
+- [While creating an order, all languages are displayed in the drop-down instead of only languages associated to the shop selected](https://github.com/PrestaShop/PrestaShop/issues/21817)
+- [When adding a product with an associated gift, the gifted product quantity is not updated](https://github.com/PrestaShop/PrestaShop/issues/21592)
+- [The gifted product is not removed when removing a product ordered](https://github.com/PrestaShop/PrestaShop/issues/21500)
+- [Edit quantity and deletion in order in BO in Multishop context : Totals are not updated](https://github.com/PrestaShop/PrestaShop/issues/21706)
+- [The Voucher "Free shipping " value is not updated when changing an address](https://github.com/PrestaShop/PrestaShop/issues/21549)
+- [Vouchers block - Voucher line is not added when a cart rule "Discount for a country selection" is applied](https://github.com/PrestaShop/PrestaShop/issues/21548)
+- [Product quantity no updated if a discount with free gift is applied](https://github.com/PrestaShop/PrestaShop/issues/21506)
+- [Recalculate shipping costs option enabled - the Shipping price not instantly updated](https://github.com/PrestaShop/PrestaShop/issues/21267)
+- [Payment block not refreshed after adding product](https://github.com/PrestaShop/PrestaShop/issues/21293)
+- [Incorrect quantity added to product when it is added to the same order as a gift also](https://github.com/PrestaShop/PrestaShop/issues/21531)
+- [Incorrect Total's product line when Round type = "Round on each item"](https://github.com/PrestaShop/PrestaShop/issues/21708)
+
+**Front-office**
+- [Pack of products - Incorrect image displayed for products with combinations](https://github.com/PrestaShop/PrestaShop/issues/21875)
+
+**Back-office**
+- [Adding incorrect language causes exception in Edit Currency page](https://github.com/PrestaShop/PrestaShop/issues/21891)
+- [Bad URL for module catalog page after disabling MBO](https://github.com/PrestaShop/PrestaShop/issues/21588)
+
+**Core**
+- [Mail translation change the product table location in html template](https://github.com/PrestaShop/PrestaShop/issues/21630)
+
+## Download
+
+You can download PrestaShop 1.7.7.0 here:
+
+[Download PrestaShop 1.7.7.0 now!](https://www.prestashop.com/en/developers-versions){: .btn .btn-lg .btn-success}
+{: .text-center}
+ 
+PrestaShop 1.7.7.0 is also available through the 1-Click Upgrade module. 
+
+
+## Acknowledgments
+
+PrestaShop is above all a community project: from the XX committers who contributed to this release, XX are new contributors. The vast majority of PrestaShop contributors are not directly affiliated with the PrestaShop company.
```

Comment:
@jolelievre Can you complete the numbers of committers and new contributors ?

Reply:
Should we mention all contributors?

Reply:
Yes, that would be great, can you list them or give me the list ?

Reply:
Yep the full list is provided in the changelog tool.